rust/bridge/jni/src/lib.rs

```diff
@@ -156,6 +156,28 @@ pub unsafe extern "C" fn Java_org_signal_client_internal_Native_IdentityKeyPair_
     })
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn Java_org_signal_client_internal_Native_IdentityKeyPair_1Deserialize(
+    env: JNIEnv,
+    _class: JClass,
+    data: jbyteArray,
+    offset: jint,
+) -> jlongArray {
+    run_ffi_safe(&env, || {
+        let offset = jint_to_u32(offset)? as usize;
```

Comment:
offset doesn't seem to be required here since it is always zero.

We support an offset for public key deserialization because Android client needed/wanted that functionality in one particular spot.

Reply:
I've removed the offset.